src/main/resources/edu/cornell/kfs/module/ar/cu-ojb-ar.xml

```diff
@@ -0,0 +1,33 @@
+<descriptor-repository version="1.0">
+
+	<class-descriptor class="org.kuali.kfs.module.ar.businessobject.ContractsGrantsInvoiceDocumentErrorLog" table="AR_INV_DOC_ERROR_LOG_T">
```

Comment:
Since the only change is increasing the width for the accounts field I don't think there are any changes to the ojb mapping for these 2 tables. If this is not needed then this file can be removed and the new entry for the databaseRepositoryFilePaths from the cu-spring-ar.xml file can be removed.